tests/ToBeScheduledTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Illuminate\Console\Scheduling\CallbackEvent;
+use Illuminate\Console\Scheduling\Event;
+use Illuminate\Support\Facades\Schedule;
+use Spatie\PestExpectations\Tests\TestSupport\Scheduled\CommandClass;
+use Spatie\PestExpectations\Tests\TestSupport\Scheduled\InvokableClass;
+use Spatie\PestExpectations\Tests\TestSupport\Scheduled\JobClass;
+
+describe('call', function () {
```

Comment:
Remove these `describe` usages. We generally don't need those, and the test names are clear enough.